framework_tvb/tvb/interfaces/rest/server/run.py

```diff
@@ -78,6 +79,7 @@ def initialize_flask():
     # creating the flask app
     app = Flask(__name__)
     app.json_encoder = CustomFlaskEncoder
+    app.jinja_env.add_extension(formencode_jinja2.formfill)
```

Comment:
I do not think for the rest api formencode-jinja will be really usefull.
I was mostly thinking of Cherrypy server when serving Jinja templates.

Reply:
Ok. I revert configurations in this file.